changelogs/unreleased/430-akhilerm

```diff
@@ -0,0 +1 @@
+add new sampling probe to detect devices used by localPV, zfs-localPV, mayastor and cstor.
```

Comment:
n_: add new discovery probe (called used-by-probe) to detect if devices are used by zfs-localPV, mayastor and cstor

Reply:
Done.